Classes/Form/AbstractFormComponent.php

```diff
@@ -12,7 +12,7 @@
 use FluidTYPO3\Flux\Form\Container\Column;
 use FluidTYPO3\Flux\Form\Container\Container;
 use FluidTYPO3\Flux\Form\Container\Grid;
-use FluidTYPO3\Flux\Form\Container\Object;
+use FluidTYPO3\Flux\Form\Container\ObjectContainer;
```

Comment:
I would suggest using `Object as ObjectContainer` and we need the very same change in the test.

Reply:
oh, my fault... of course!